ParisBNF/abb/BNFabb156.xml

```diff
@@ -155,177 +158,181 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                      <textLang mainLang="gez"/>                     
                   </msItem>
                   
-                  <msItem xml:id="ms_i3.7">
+                  <msItem xml:id="ms_i4.7">
                      <locus from="95vb" to="97rb"/>
                      <title type="complete"  ref="LIT2057Bookof">Book of Nahum</title>
                      <textLang mainLang="gez"/>                     
                   </msItem>
                   
-                  <msItem xml:id="ms_i3.8">
+                  <msItem xml:id="ms_i4.8">
                      <locus from="97rb" to="98va"/>
                      <title type="complete"  ref="LIT1567Bookof">Book of Habbakuk</title>
                      <textLang mainLang="gez"/>                     
                   </msItem>
                   
-                  <msItem xml:id="ms_i3.9">
+                  <msItem xml:id="ms_i4.9">
                      <locus from="98va" to="99vb"/>
                      <title type="complete"  ref="LIT3148Zephan">Book of Zephania</title>
                      <textLang mainLang="gez"/>                     
                   </msItem>
                   
-                  <msItem xml:id="ms_i3.10">
+                  <msItem xml:id="ms_i4.10">
                      <locus from="99vb" to="101ra"/>
                      <title type="complete"  ref="LIT3149Haggai">Book of Haggai</title>
                      <textLang mainLang="gez"/>                     
                   </msItem>
                   
                   
-                  <msItem xml:id="ms_i3.11">
+                  <msItem xml:id="ms_i4.11">
                      <locus from="101ra" to="106vb"/>
                      <title type="complete"  ref="LIT3150Zechar">Book of Zechariah</title>
                      <textLang mainLang="gez"/>                     
                   </msItem>
                   
-                  <msItem xml:id="ms_i3.12">
+                  <msItem xml:id="ms_i4.12">
                      <locus from="106vb" to="108rb"/>
                      <title type="complete" ref="LIT3151Malachi">Book of Malachi</title>
                      <textLang mainLang="gez"/>                     
                   </msItem>
                   </msItem>
                   
                   
-                  <msItem xml:id="ms_i4">
+                  <msItem xml:id="ms_i5">
                      <locus from="108va" to="130vb"/>
                      <title type="complete" ref="LIT3183Commen">Commentaries on the Books of the Minor Prophets, in Gǝʿǝz</title>
                   <textLang mainLang="gez"/>                     
                   
              
-                  <msItem xml:id="ms_i4.1">
+                  <msItem xml:id="ms_i5.1">
                      <locus from="108va" to="111vb"/>
                      <title type="complete" ref="LIT3183Commen#Hosea">Interpretation of the kings' names mentioned in Hosea</title>
```

Comment:
Is the relevant structure (IDs labels for DIVs) provided in `LIT3183`? I cannot find any chapters

Reply:
What do you mean? The structure is under LIT3183

Reply:
I see, the latest version has not made it to the app (no structure visible here https://betamasaheft.eu/works/LIT3183Commen/main), will re-expand manually